tests/vcr_python_wrapper.py

```diff
@@ -10,17 +10,17 @@
 # We need our own json level2 matcher, because, python2 and python3 do not save
 # dictionaries in the same order
 def body_json_l2_matcher(r1, r2):
-    if r1.headers.get('content-type') == 'application/json' and r2.headers.get('content-type') == 'application/json':
+    if r1.headers.get('content-type') == r2.headers.get('content-type') == 'application/json':
         if r1.body is None or r2.body is None:
             return r1.body == r2.body
         body1 = json.loads(r1.body.decode('utf8'))
         body2 = json.loads(r2.body.decode('utf8'))
-        if 'search' in body1:
```

Comment:
I don't quite understand this hunk.
The old code would sort our `search` param. The new code does not touch `search` at all? And it still works?

Granted, apypie passes the search in the URL, not in the body, but I still don't understand why this doesn't break the nailgun based modules.

Reply:
That is due to the way nailgun delivered the search param. Now we have no nailgun module left, that searches with >= 2 parameters. So sorting is a noop.
In Apypie the search is no longer part of the body, but in the querystring.

Reply:
I wished so hard that the sentence would end at "Now we have no nailgun module left" :rofl: